sites/game-client/client.css

```diff
@@ -34,6 +39,38 @@ body {
   box-sizing: content-box;
 }
 
+#avatar-selection-div {
+  position: absolute;
+  top: 0px;
+  left: 0px;
+  width: 100vw;
+  height: 100vh;
+  box-sizing: border-box;
+  z-index: 100;
```

Comment:
Maybe 99 or 90 so error modal will be above it?

Reply:
fixed